force-app/main/default/classes/ESA_cls_ExternalCommitteeTriggerHandler.cls

```diff
@@ -0,0 +1,61 @@
+/**********************************************************************************************
+* @Author:      Vasanthi D 
+* @Date:        11 Sep 2023
+* @Description: The purpose of this class is to create methods which can be used by different trigger event
+* @Revision(s): [Date] - [Change Reference] - [Changed By] - [Description]   
+***********************************************************************************************/
+public with sharing class ESA_cls_ExternalCommitteeTriggerHandler {
+    public static boolean cloneOpenCases_runsOne = TRUE;
+    public static void cloneOpenCases(List<External_Committee__c> newMeetings){
+        cloneOpenCases_runsOne = FALSE;
+        set<id> parentIds  = new set<id>();
+        Map<id,list<Case>> openCaseMap = new  Map<id,list<Case>>();
+        for(External_Committee__c ec : newMeetings){
+            if(ec.isClone()){
+                parentIds.add(ec.getCloneSourceId());
+            }
+        }     
+        List<Case> caseList = new List<Case>([select id,External_Committee__c,Status from Case where External_Committee__c IN:parentIds AND Status  IN ('Received' ,'Under Review') ]);
```

Comment:
@deepakmulamalla Based on my other comments, this SOQL needs to have uniform use of SELECT, FROM, WHERE and AND.

Reply:
@deepakmulamalla : What if `caseList` list returns empty? the below iteration / looping is done without checking for empty list.

Reply:
The SOSL has been changed to upper-case as suggested.